dll/keyboard/kbdsf/kbdsf.c

```diff
@@ -0,0 +1,479 @@
+/*
+ * ReactOS French (Switzerland) ASCII Keyboard layout
+ * Copyright (C) 2005 ReactOS
+ * License: LGPL, see: LGPL.txt
```

Comment:
As you are copying the header from neighbor files, I noticed the Author field is missing, please fill it in. Also suggesting to specify current year for copyright.
```suggestion
 * Copyright (C) 2020 ReactOS
 * Author: ???
 * License: LGPL, see: LGPL.txt
```